r2_0.86

tajinderpalsingh61 opened the pull request:
Feature Engineering for Machine Learning Assignment

test mse: 923125269.4375038
test rmse: 30382.97663885986
test r2: 0.8656708697494032

tajinderpalsingh61 committed:
r2_0.86

solegalli commented:
Hi @tajinderpalsingh61 

Some of the cells of the notebook are not run, so it is not possible to see if your model has a higher performance. Could you upload a new version where we can see the output of all cells?

tajinderpalsingh61 committed:
Feature Engineering Assignment

test mse: 923125269.4375038
test rmse: 30382.97663885986
test r2: 0.8656708697494032

tajinderpalsingh61 commented:
@solegalli Ok, Kindly refer https://github.com/solegalli/udemy-feml-challenge/pull/35/commits/9559fd3c8c89e0ca29fd5e6a32937880d998cf22

solegalli commented:
@tajinderpalsingh61 well done! Thank you!
